docker/build_scripts/requirements3.10.txt

```diff
@@ -32,9 +32,9 @@ wheel==0.37.1 \
     # via -r requirements.in
 
 # The following packages are considered to be unsafe in a requirements file:
-pip==22.0 \
-    --hash=sha256:6cb1ea2bd7fda0668e26ae8c3e45188f301a7ef17ff22efe1f70f3643e56a822 \
-    --hash=sha256:d3fa5c3e42b33de52bddce89de40268c9a263cd6ef7c94c40774808dafb32c82
+pip==22.0.1 \
```

Comment:
@mayeut - unfortunately this version has an issue with how it handles non PyPI package indexes. Is there any chance we could bump this version to ``22.0.2``?  https://pip.pypa.io/en/stable/news/#v22-0-2

Reply:
This is done.
FYI, if you don't already know this, each commit is tagged in quay.io so these problems can be worked around by getting back to the version that worked for you: https://quay.io/repository/pypa/manylinux2014_x86_64?tab=tags

Reply:
Wonderful, thank you.